doto-frontend/src/components/pages/Calendar/CalendarComponent.js

```diff
@@ -111,6 +115,11 @@ export function Content({
 
     return (
         <AppointmentTooltip.Content {...restProps} appointmentData={appointmentData}>
+            <Grid container alignItems="center">
+                <div className={classes.dueContainer}>
+                    <div>Due: {new Date(appointmentData.dueDate).toLocaleString()}</div>
```

Comment:
```suggestion
                    <div>Due: {new Date(appointmentData.dueDate).toLocaleString("en-nz")}</div>
```

Reply:
AFAIK, This should be done already when populating `dueDate` in the DB (In other controllers). However, will add this in just for sanity check.